cmd/infra/azure/create.go

```diff
@@ -201,14 +201,14 @@ func (o *CreateInfraOptions) Run(ctx context.Context, l logr.Logger) (*CreateInf
 	} else {
 		//create a resource group for virtual network
 		vnetResourceGroupName = o.Name + "-vnet"
-		vnetRG, msg, err := createResourceGroup(ctx, o, azureCreds, vnetResourceGroupName, subscriptionID)
+		vnetResourceGroupName, msg, err = createResourceGroup(ctx, o, azureCreds, vnetResourceGroupName, subscriptionID)
```

Comment:
Similar comment as above

Reply:
nope when we create a resource group name we also have a unique identifier as a suffix